app/experimenter/experiments/constants/nimbus.py

```diff
@@ -45,8 +39,8 @@ class NimbusTargetingConfig:
         "locale using the release channel."
     ),
     targeting=(
-        "localeLanguageCode == 'en' && (isFirstStartup || '{slug}' in "
-        "activeExperiments) && browserSettings.update.channel == '{firefox_channel}'"
+        "localeLanguageCode == 'en' "
+        "&& (isFirstStartup || experiment.slug in activeExperiments)"
```

Comment:
Heh, that didn't take long for an update on how the targeting works. 😉

Reply:
;)